<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>print_basedialog</class>
 <widget class="QDialog" name="print_basedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printing selection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="PixmapLabel2">
     <property name="pixmap">
      <pixmap>:/gex/icons/print.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Pages to print</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="RadioCurrentPage">
        <property name="toolTip">
         <string>Print current page</string>
        </property>
        <property name="text">
         <string>Current page</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="RadioReport">
        <property name="toolTip">
         <string>Print multiple pages at once!</string>
        </property>
        <property name="text">
         <string>Report sections:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="CheckGlobal">
        <property name="toolTip">
         <string>Print global information section</string>
        </property>
        <property name="text">
         <string>Global info</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="CheckWafermap">
        <property name="toolTip">
         <string>Print Wafer map section</string>
        </property>
        <property name="text">
         <string>Wafer map</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="CheckAdvanced">
        <property name="toolTip">
         <string>Print Binning section</string>
        </property>
        <property name="text">
         <string>Advanced reports</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="CheckStats">
        <property name="toolTip">
         <string>Print Test Statistics section</string>
        </property>
        <property name="text">
         <string>Test Statistics</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="CheckBinning">
        <property name="toolTip">
         <string>Print Binning section</string>
        </property>
        <property name="text">
         <string>Binning</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="CheckSnapshotGallery">
        <property name="toolTip">
         <string>Print Binning section</string>
        </property>
        <property name="text">
         <string>Snapshots Gallery</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="CheckHistograms">
        <property name="toolTip">
         <string>Print Histograms section</string>
        </property>
        <property name="text">
         <string>Histograms</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="CheckPareto">
        <property name="toolTip">
         <string>Print Binning section</string>
        </property>
        <property name="text">
         <string>Pareto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Notes:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTextEdit" name="TextEditComment"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>250</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="TextLabel1">
     <property name="text">
      <string>Smart HTML printing (print multiple sections at once)!</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="PushButtonOk">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Next &gt;&gt;</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>RadioCurrentPage</tabstop>
  <tabstop>RadioReport</tabstop>
  <tabstop>CheckGlobal</tabstop>
  <tabstop>CheckStats</tabstop>
  <tabstop>CheckHistograms</tabstop>
  <tabstop>CheckWafermap</tabstop>
  <tabstop>CheckBinning</tabstop>
  <tabstop>CheckPareto</tabstop>
  <tabstop>CheckAdvanced</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
